libdrm/plan.sh

```diff
@@ -0,0 +1,24 @@
+pkg_name=libdrm
+pkg_origin=core
+pkg_version=2.4.88
+pkg_description="Direct Rendering Manager"
+pkg_upstream_url="https://dri.freedesktop.org/wiki/"
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_license=('MIT')
+pkg_source="https://dri.freedesktop.org/${pkg_name}/${pkg_name}-${pkg_version}.tar.gz"
+pkg_shasum=a8b458db6a73c717baee2e249d39511fb6f5c0f5f24dee2770935eddeda1a017
+pkg_deps=(
+  core/libpciaccess
+  core/glibc
+)
+pkg_build_deps=(
+  core/cairo   # not linked by bins/libs
+  core/diffutils
+  core/gcc
+  core/libxslt # not linked by bins/libs
```

Comment:
No worries about this not being linked.  It is likely just to get the xsltproc binary.  My guess is it is used when building the manpages perhaps.